mysite/fsm/models.py

```diff
@@ -82,78 +41,106 @@ def save_json_data(self, d=None, attr='data', doSave=True):
 )
 
 
-def load_json_id(name, pk):
+class JSONBlobMixin(object):
```

Comment:
@maxsocl I think the only methods that belong in this mixin are the four that actually *are* methods: load_json_data(), save_json_data(), get_data_attr(), and set_data_attr().  The others do not even use self or cls, and should be left as functions: dump_json_id(), dump_json_id_dict(), load_json_id(), load_json_id_dict().